reo/process_results.py

```diff
@@ -695,13 +198,21 @@ def __init__(self, results_dict, results_dict_bau, dm, inputs):
             self.nested_outputs = self.setup_nested()
 
         @property
-        def replacement_costs(self):
-            replacement_costs = 0
-            replacement_costs += self.inputs["Storage"]["replace_cost_us_dollars_per_kw"] * \
+        def replacement_costs_future_and_present(self):
+            future_cost_inverter = self.inputs["Storage"]["replace_cost_us_dollars_per_kw"] * \
                                  self.nested_outputs["Scenario"]["Site"]["Storage"]["size_kw"]
-            replacement_costs += self.inputs["Storage"]["replace_cost_us_dollars_per_kwh"] * \
+            future_cost_storage = self.inputs["Storage"]["replace_cost_us_dollars_per_kwh"] * \
                                  self.nested_outputs["Scenario"]["Site"]["Storage"]["size_kwh"]
-            return round(replacement_costs, 2)
+            future_cost = future_cost_inverter + future_cost_storage
+
+            tax_rate = self.inputs["Financial"]["owner_tax_pct"]
+            discount_rate = self.inputs["Financial"]["owner_discount_pct"]
+            present_cost = 0
+            present_cost += future_cost_inverter * (1 - tax_rate) / ((1 + discount_rate) **
```

Comment:
Not critical, but is apply the 1-tax_rate to future capital costs the correct way to calculate after tax?